lib/Default/Plotter.class.inc

```diff
@@ -17,8 +17,18 @@ class Plotter {
 					return SmrLocation::getLocation($X);
 				}
 				return $X;
-			case 'Goods':
```

Comment:
It might be worth leaving the generic option in

Reply:
I was having trouble thinking of instances where you want to find a good, but don't care if the port buys or sells it. Do you have any in mind?

Reply:
The only thing that comes to mind would be route building, but it would be crazy to try doing that via the in-game plot nearest, so nvm